website/blog/2020-06-16-alluxio-datacaching.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Improving Presto Latencies with Alluxio Data Caching
+author: Rohit Jain
+authorURL: https://www.linkedin.com/in/jain-rohit/
+---
+
+**Facebook:** Rohit Jain, James Sun, Ke Wang, Shixuan Fan, Biswapesh Chattopadhyay, Baldeep Hira
+
+**Alluxio:** Bin Fan, Calvin Jia, Haoyuan Li
+
+
+# Introduction
+
+The Facebook Presto team has been collaborating with [Alluxio](https://www.alluxio.io/) on an open source data caching solution for Presto. 
+This is required for multiple Facebook use-cases to improve query latency for queries that scan data from remote sources, such as HDFS. 
+We have observed significant improvements in query latencies and IO scans in our experiments. 
+<br><br>
+We found Alluxio data caching to be useful for multiple use-cases in the Facebook environment. 
+For one of the Facebook internal use cases, we observed query latencies improved by **33%** (P50), **54%** (P75), and **48%** (P95). 
+We also recorded **57%** improvement in IO for remote data source scans.
+
+# Presto Architecture
```

Comment:
Add the truncate tag before this so the blog index doesn't show the whole post

Reply:
It wasn't really showing it.
I anyways think the blog index is broken, it does not show H1 titles. Any idea about it?
Even Uber blog has broken indexes.

Reply:
Sorry, I confused two things.
Blog index does not show the whole page now.
However the table of contents on each blog(top right section) does not seem to show all headers.